### PR TITLE
New functions: acos(), find(), and getLengthX()

### LINKED ISSUE #56: Improvements to SVG functions
Here are below some notes from using the SVG functions in a real-world use case.

* Path normalization could slightly be improved by removing all `M` commands that are not at the beginning of the path. This happens when concatenating paths together for instance. This can easily be fixed at the user level but since we claim to support **any** SVG path, it might be useful to be more 

* Implement `solveCubicRootForBezier()`. This is a tricky one. Not only it would be a very complex algorithm to reimplement in Reanimated (here is a good JS implementation: https://github.com/tab58/minimatrix-polyroots/blob/master/index.js#L133), we would also need to add the root selection (real number between 0 and 1).

### PR DESCRIPTION
**Pull request by wcandillon**
fixes #56

### CODE REVIEW

**Comment by wcandillon** [comment]
@dsznajder These are functions that I build for a project of mine, it works pretty well :)

**Comment by dsznajder** [comment]
@wcandillon I saw it ❤️ !

File: src/Array.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Animated from "react-native-reanimated";
 
 const { Value, cond, eq, or } = Animated;
 
-export const find = (
+export const get = (
```
Comment (dsznajder) [inline src/Array.ts]: Breaking change?

File: src/bezier/CubicBezierSolve.ts [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  const crtr: Animated.Value<number> = new Value();
+  const t1: Animated.Value<number> = new Value();
+
+  const u1: Animated.Value<number> = new Value();
+
+  const sd: Animated.Value<number> = new Value();
+  const v1: Animated.Value<number> = new Value();
+  const sq: Animated.Value<number> = new Value();
+
+  return block([
```
Comment (dsznajder) [inline src/bezier/CubicBezierSolve.ts]: 👾 👾 👾

File: src/Array.ts (hunk 1 above)
Comment (wcandillon) [reply src/Array.ts]: yes! but that makes sense right?

File: src/bezier/CubicBezierSolve.ts (hunk 2 above)
Comment (wcandillon) [reply src/bezier/CubicBezierSolve.ts]: I copied it line by line from https://pomax.github.io/bezierinfo/#quartic-curves-cardano-s-algorithm- works super well :) btw this page on bezier curves is amazing: https://pomax.github.io/bezierinfo. I learned so much from it, it's crazy

Comment (dsznajder) [reply src/bezier/CubicBezierSolve.ts]: I will dive into it through the nearest weekend, right after trying hard with `loop` :D

Comment (wcandillon) [reply src/bezier/CubicBezierSolve.ts]: haha `loop`, `toggle`,  and `delay`, are the real hard stuff :) I'm just distracting myself in between with some math stuff 😅🙈

**Pull request merged**